spec/models/assignment_spec.rb

```diff
@@ -2227,6 +2227,42 @@ def grouping_count(groupings)
     end
   end
 
+  describe '#get_num_valid' do
+    before :each do
+      @assignment = create(:assignment, assignment_properties_attributes: { group_min: 2, group_max: 3 })
+      @groupings = Array.new(4) { create(:grouping, assignment: @assignment) }
+    end
+
+    context 'When two groups meet min size requirement' do
```

Comment:
start these labels with lowercase letters rather than uppercase (this is just a style convention, and is useful when nesting `context` blocks)

(fix in cases below as well)